Corona-Warn-App/src/main/java/de/rki/coronawarnapp/coronatest/server/VerificationApiV1.kt

```diff
@@ -1,21 +1,21 @@
 package de.rki.coronawarnapp.coronatest.server
 
-import com.google.gson.annotations.SerializedName
+import com.fasterxml.jackson.annotation.JsonProperty
 import retrofit2.http.Body
 import retrofit2.http.Header
 import retrofit2.http.POST
 
 interface VerificationApiV1 {
 
     data class RegistrationTokenRequest(
-        @SerializedName("keyType") val keyType: VerificationKeyType,
-        @SerializedName("key") val key: String,
-        @SerializedName("keyDob") val dateOfBirthKey: String? = null,
-        @SerializedName("requestPadding") val requestPadding: String? = null,
+        @JsonProperty("keyType") val keyType: VerificationKeyType,
+        @JsonProperty("key") val key: String,
+        @JsonProperty("dateOfBirthKey") val dateOfBirthKey: String? = null,
```

Comment:
should not this be `"keyDob"` ? how come it changes to `"dateOfBirthKey"` ?

Reply:
Switched back to keyDob